SKYKitChat/Classes/UI/ParticipantList/SKYChatParticipantListViewController.swift

```diff
@@ -208,6 +213,7 @@ extension SKYChatParticipantListViewController: UITableViewDelegate, UITableView
         if let d = self.delegate {
             let participant = self.participants[indexPath.row]
             d.listViewController?(self, didSelectParticipant: participant)
+            self.dismiss(animated: true)
```

Comment:
1. I don’t know why this is suddenly here, the commit doesn’t seem related to view controller flow.
2. Suppose the user of this view controller class did not specify a delegate, you should still dismiss this view controller anyway (if dismissing is the intended behavior).